lib/symboldatabase.cpp

```diff
@@ -1279,6 +1280,102 @@ void SymbolDatabase::createSymbolDatabaseEnums()
     }
 }
 
+void SymbolDatabase::createSymbolDatabaseIncompleteVars()
+{
+    static const std::set<std::string> keywords = {
```

Comment:
sorry but the list of keywords has to depend on which standard is used.

And we avoid static containers like this in the functions for thread safety reasons. Remove the `static` keyword or move out the set.

Reply:
I separated the keywords.